.github/workflows/ms.servicefabric.clusters.yml

```diff
@@ -0,0 +1,214 @@
+name: 'Service Fabric: Clusters'
+
+on:
+  workflow_dispatch:
+    inputs:
+      removeDeployment:
+        description: 'Remove deployed module'
+        required: false
+        default: 'true'
+      versioningOption:
+        description: 'The mode to handle the version increments [major|minor|patch|custom]'
+        required: false
+        default: 'patch'
+      customVersion:
+        description: 'The version to enforce if [versionOption] is set to [custom]'
+        required: false
+        default: '0.0.1'
+  push:
+    branches:
+      - main
+    paths:
+      - '.github/actions/templates/**'
+      - '.github/workflows/ms.servicefabric.clusters.yml'
+      - 'arm/Microsoft.ServiceFabric/clusters/**'
+      - '!arm/Microsoft.ServiceFabric/clusters/readme.md'
+
+env:
+  moduleName: 'clusters'
+  modulePath: 'arm/Microsoft.ServiceFabric/clusters'
+  workflowPath: '.github/workflows/ms.servicefabric.clusters.yml'
+  AZURE_CREDENTIALS: ${{ secrets.AZURE_CREDENTIALS }}
+  ARM_SUBSCRIPTION_ID: ${{ secrets.ARM_SUBSCRIPTION_ID }}
+  ARM_MGMTGROUP_ID: ${{ secrets.ARM_MGMTGROUP_ID }}
+
+jobs:
+  ############################
+  #   SET INPUT PARAMETERS   #
+  ############################
+  job_set_workflow_param:
+    runs-on: ubuntu-20.04
+    name: 'Set input parameters to output variables'
+    steps:
+      - name: 'Checkout'
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: 'Set input parameters'
+        id: get-workflow-param
+        uses: ./.github/actions/templates/getWorkflowInput
+        with:
+          workflowPath: '${{ env.workflowPath}}'
+    outputs:
+      removeDeployment: ${{ steps.get-workflow-param.outputs.removeDeployment }}
+      versioningOption: ${{ steps.get-workflow-param.outputs.versioningOption }}
+      customVersion: ${{ steps.get-workflow-param.outputs.customVersion }}
+
+  ##################
+  #   UNIT TESTS   #
+  ##################
+  # Global tests
+  # ------------
+  job_tests_module_global:
+    runs-on: ubuntu-20.04
+    name: 'Run global module tests'
+    steps:
+      - name: 'Checkout'
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: 'Test module'
+        uses: ./.github/actions/templates/validateModuleGeneral
+        with:
+          modulePath: '${{ env.modulePath }}'
+          moduleName: '${{ env.moduleName}}'
+
+  # Global API tests
+  # ----------------
+  job_tests_module_global_api:
+    runs-on: ubuntu-20.04
+    name: 'Run global API module tests'
+    steps:
+      - name: 'Checkout'
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: 'Test module'
+        uses: ./.github/actions/templates/validateModuleApis
+        with:
+          modulePath: '${{ env.modulePath }}'
+          moduleName: '${{ env.moduleName}}'
+
+  # Validate deployment module tests
+  # --------------------------------
+  job_tests_module_deploy_validate:
```

Comment:
Please update to latest pipeline version. This one is out of date. We actually merged the deploy_validate tests into the validate action. Please replace

Reply:
Updated.

Reply:
Again outdated. Please update